files/image_config/logrotate/logrotate-rsyslog.j2

```diff
@@ -30,8 +30,15 @@
 /var/log/telemetry.log
 /var/log/quagga/bgpd.log
 /var/log/quagga/zebra.log
+{% if namespaces > 1 %}
+{% for ns in range(namespaces) %}
+/var/log/swss{{ns}}/sairedis.rec
+/var/log/swss{{ns}}/swss.rec
```

Comment:
I am curious about other logs such as quagga, teamd? they are also running in the each namespace? are we aggregating all of them into one log file?

Reply:
i am not fan using templating everywhere. I think a better solution here is to add option to both swss and sairedis to allow specify the log file name, and we can use swss.asic{n}.rec, sairedis.asic{n}.rec. then, we do not need to use template for rsyslog file. 

In the future, if we need to dynamically incrase the asic number of the platform, there is no need to change this file.